src/vaadin-crud-grid.html

```diff
@@ -43,12 +43,16 @@
             /**
              * Do not add headers to columns.
              */
-            noHead: Boolean
+            noHead: Boolean,
+            hideEditColumn: Boolean
```

Comment:
Not sure, but I guess this one can be private.

Reply:
Now it's private.